test/geocoders/teleport.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: UTF-8 -*-
+from geopy.geocoders import Teleport
+from test.geocoders.util import GeocoderTestBase
+
+
+class TeleportTestCaseUnitTest(GeocoderTestBase):
+
+    def test_user_agent_custom(self):
+        geocoder = Teleport(
+            user_agent='my_user_agent/1.0'
+        )
+        self.assertEqual(geocoder.headers['User-Agent'], 'my_user_agent/1.0')
+
+
+class TeleportTestCase(GeocoderTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.delta = 0.04
+
+    def test_unicode_name(self):
+        """
+        Teleport.geocode unicode
+        """
+        # work around ConfigurationError raised in GeoNames init
+        self.geocoder = Teleport(scheme='http')
```

Comment:
I think of switching all tests to https. Almost all of them (if not exactly all) support https these days. So if this is the case for Teleport, could you please remove the `scheme` kwarg, to use the default (`https`) scheme?